task/02-numbers-tasks.js

```diff
@@ -54,7 +54,14 @@ function getCicleCircumference(radius) {
  *  -3, 3  => 0
  */
 function getAverage(value1, value2) {
-    throw new Error('Not implemented');
+    const sum = value1 + value2;
+    if(sum > Number.MAX_VALUE){
```

Comment:
`sum > Number.MAX_VALUE` very dangerous place. Sum is a number and how can it be more than `Number.MAX_VALUE` if it is a max value :)?